client/app/components/Parameters.less

```diff
@@ -7,29 +7,50 @@
   background-size: 2px 2px;
   display: inline-block;
   width: 6px;
-  height: 34px;
+  height: 36px;
   vertical-align: bottom;
   margin-right: 5px;
-  margin-bottom: 2px;
   cursor: move;
 }
 
 .parameter-block {
   display: inline-block;
   background: white;
-  padding: 6px;
+  padding: 3px 6px 6px;
+
+  .parameter-container[data-draggable] & {
+    margin: 4px 0 0 4px;
+  }
+
+  // dragged element
+  body > & {
```

Comment:
No other hook 😅

Reply:
It is possible to use `helperContainer` attr to append it somewhere else than the body, but I don't think it's worth it :)

Reply:
I wonder if `helperClass` is targeted on that element, I'll check that and edit this comment with my conclusion.

Reply:
aaad44e